src/ORM/Connect/Database.php

```diff
@@ -246,6 +246,16 @@ public function isActive()
     }
 
     /**
+     * The identifier quote char. For ANSI this is " and usually is ` for others
+     *
+     * @return string
+     */
+    public function getIdentifierQuoteCharacter()
+    {
+        return '"';
```

Comment:
As I said above, we need to fix this to `"` for now. We aren't able to yet soft-code this.